pages/content/ch2/propositional-logic.mdx

```diff
@@ -50,41 +50,24 @@ $\vDash$
-       & \quad \vert \; \text{"} \to \text{"} \\
-       & \quad \vert \; \text{"} \lnot \text{"} \\
-\\  
-\textit{FormulaToStatementOperator} & \Leftarrow \text{"} \equiv \text{"} \\
-       & \quad \vert \; \text{"} \models \text{"} \\
-\\
-\textit{StatementOperator} & \Leftarrow \text{"} \implies \text{"} \\
-       & \quad \vert \; \text{"} \iff \text{"} \\
-       & \quad \vert \; \text{"and"} \\
-       & \quad \vert \; \text{"or"} \\
-\\   
-\textit{Formula} & \Leftarrow \textit{Symbol} \\
-       & \quad \vert \; \textit{Symbol } \textit{ LogicalOperator } \textit{ Symbol} \\
-       & \quad \vert \; "("\textit{Formula } \textit{ LogicalOperator } \textit{ Formula}")" \\
-       & \quad \vert \; \text{"} \top \text{"} \\
-       & \quad \vert \; \text{"} \bot \text{"} \\
-\\   
-\textit{Statement} & \Leftarrow \textit{Symbol} \\
-       & \quad \vert \; \textit{Statement } \textit{ StatementOperator } \textit{ Statement} \\
-       & \quad \vert \; \textit{Statement } \text{ "is false"} \\
-       & \quad \vert \; \textit{Formula } \textit{ FormulaToStatementOperator } \textit{ Formula} \\
-       & \quad \vert \; \text{"} \models \text{"} \textit{ Formula} \\
-\end{aligned}
-$$
-
+<div className="mt-4 [&_code]:font-[inherit] [&_code]:italic">
+
+  This tries to define formulas and statements with EBNF. Bold symbols are used to denote EBNF syntax elements **(** **)** **\[** **]** **│**. References to existing rules are made using `Rulename…` element.
+
+  | Rule                | Expression                                                                                                 |
+  |---------------------|------------------------------------------------------------------------------------------------------------|
+  | `PropSymbol`        | $A$ **│** $B$ **│** $C$ **│** ••• **│** $Z$                                                                |
+  | `Symbol`            | `PropSymbol` **│** $\top$ **│** $\bot$                                                                     |
+  | `LogicOperator`     | $\land$ **│** $\lor$ **│** $\leftrightarrow$ **│** $\to$                                                   |
+  | `Negation`          | $\lnot$                                                                                                    |
+  | `Combination`       | **(**`Formula` **│** $($`Combination`$)$**)** `Operator` **(**`Formula` **│** $($`Combination`$)$**)**     |
+  | `Formula`           | **[**`Negation`**]**`Symbol` **│** `Combination` **│** `Negation`$($`Combination`$)$                       |
+  | `FormulaStatement`  | `Formula` **(** $\equiv$ **│** $\models$ **)** `Formula` **│** $\models$ `Formula`                         |
+  | `StatementOperator` | $\implies$ **│** $\iff$ **│** $\text{ and }$ **│** $\text{ or }$                                           |
+  | `Proposition`       | `FormulaStatement` **│** `Statement` **│** `PropSymbol`&nbsp;or&nbsp;mathematical&nbsp;statement           |
```

Comment:
is this line necessary (`Statement` should be enough?)? And what exactly is "`PropSymbol` or mathematical statement"?

Reply:
I thought of something like `All prime numbers are odd` (this would be a "mathematical statement"; a proposition, just prose text) and then a a statement could be "`All prime numbers are odd` is false"

I tried to split the statement definition into logical parts to prevent strange combinations. We also had examples like "$S$ is false" thus my attempt to prevent incompatible combinations.
Also this way, the cell content is not on multiple lines.

Reply:
but shouldn't we use `**|**` instead of `or`?

Reply:
Yeah. I just thought the prose could also contain the propositional symbols like A or S

Reply:
Also "Mathematical statement" is maybe not clear enough?

Reply:
> Also "Mathematical statement" is maybe not clear enough?

should be fine, this is also used in e.g. Def. 2.1